Docs/updategems.md

```diff
@@ -0,0 +1,10 @@
+# UPDATE GEM PACKAGES TO AVOID VULNERABILITIES 
+
+From time to time it is important to update the version of the gems we are working with in our project because having outdated versions can generate serious security vulnerabilities.
+
+In order to update the version of the packages we are going to execute the following commands.
+
+- export JEKYLL_VERSION=4.2.0 (This is to set the "JEKYLL_VERSION" variable value. You can check this value using "echo JEKYLL_VERSION").
```

Comment:
Although it is kind of obvious, we could add here that they can check the right version to choose in the docker file.